ft_sourceplot.m

```diff
@@ -327,6 +327,33 @@ function ft_sourceplot(cfg, functional, anatomical)
   tmpcfg.parameter = {cfg.funparameter, cfg.maskparameter, cfg.anaparameter};
   functional = ft_volumedownsample(tmpcfg, functional);
   [cfg, functional] = rollback_provenance(cfg, functional);
+elseif hasanatomical && strcmp(cfg.method, 'cloud') % need to disentangle the volumetric from the surface inputs
+  cloudsurf = {};
+  cloudvol = {};
+  
+  % check that anatomical is a cell array
+  if ~iscell(anatomical)
```

Comment:
the field "anatomical" is a single anatomical MRI or a single structure with mesh information, such as a cortical sheet. It is not (and cannot be) a cell array: that is not how FT works.